Services/Infrastructure/OJS.Services.Infrastructure/Cache/Implementations/CacheService.cs

```diff
@@ -21,76 +21,87 @@ public class CacheService : ICacheService
         private readonly IMemoryCache memoryCache;
         private readonly IEmailService emailService;
         private readonly EmailServiceConfig emailConfig;
+        private readonly IConnectionMultiplexer redisConnection;
+        private readonly string defaultSubject = "RedisConnection";
```

Comment:
You can move these variables to Constants.cs

Reply:
Done